src/commands/set.js

```diff
@@ -21,7 +21,7 @@ import { writeJSONSync } from '../utils/fs';
 import { createCommand } from '../utils/helpers';
 
 const availableVariables = CONFIG_VARIABLES.join(', ');
-const description = `Sets configuration <variable> to <value>. Variables available: ${availableVariables}. Configuration is persisted in \`${configFilePath}\`.
+const description = `Sets configuration <variable> to <inputs...>. Variables available: ${availableVariables}. Configuration is persisted in \`${configFilePath}\`.
```

Comment:
Maybe this should be `Sets configuration <key> to [value(s)...].`? (Also applies to command configuration below.)

Reply:
When we have multiple inputs, it's using `inputs`, so I think it's better to be consistent

Reply:
But `variable` is also an input?